packages/pagebuilder/lib/handleHtmlContentClick.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Helper function for onClick() HTML Events
+ *
+ * @param {object} history history object
+ * @param {function} history.push Pushes a new entry onto the history stack
+ * @param {Event} event
+ */
+const handleHtmlContentClick = (history, event) => {
+    const { target } = event;
+    // Intercept link clicks and check to see if the
+    // destination is internal to avoid refreshing the page
+    if (target.tagName === 'A') {
```

Comment:
is anchor tag not lowercase?

Reply:
yes, they are always capitalized in this case since we're working with HTML DOM trees

https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName#value